### DISCUSSION TITLE
Could not load file or assembly 'NuGet, Version=6.9.1.32767, Culture=neutral, PublicKeyToken=' when running nuget from source code

### DISCUSSION THREAD

**Discussion by DmitryLukyanov** [NuGet/Home#13900] [General]
I've checked out the nuget sources from git (https://github.com/NuGet/NuGet.Client - v6.9.1.3) to investigate a bug that is described in https://github.com/NuGet/Home/issues/13899

The error mentioned there happens in `ProjectFactory.ProcessDependencies` method in `Nuget.CommandLine` project. So, I want to debug it. I unloaded few projects like tests, VS related. Now I can build the remaining projects including `Nuget.CommandLine`. But when I run the project it fails with the error:
>Could not load file or assembly 'NuGet, Version=6.9.1.32767, Culture=neutral, PublicKeyToken=' when running nuget from source code

any ideas how to fix it would be greatly appreciated?

**Comment by zivkan** [comment]
Did you run the steps documented in our contribution guide? https://github.com/NuGet/NuGet.Client/blob/dev/CONTRIBUTING.md#build

In particular, you need to run `build.ps1` because the Nuget.Client repo doesn't have access to the private keys for the strong name signing that happens, and as a result, .NET Framework doesn't trust the delay signed assemblies. `build.ps1` changes machine state to ignore strong name sign verification for the two keys that the repo uses.

**Reply by DmitryLukyanov** [reply]
yeah, that works, thanks!
